### PR TITLE
Update tech radar information

### PR DESCRIPTION
**Pull request by afreire-laptop**
This PR updates Tech Radar:
 - Move technologies to the corresponding current state (ring)
 - Fix technologies category (quadrant)
 - Bump one transitive dependency (which was breaking the run)
 - Downgrade one transitive dependency (which was breaking the data parsing to markdown feature. This should be considered a temp. fix)
 
 
Screenshot 2024-12-10 at 15 28 41

### CODE REVIEW

File: docs/data/azure-api-management/azure-api-management.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 label: Azure API Management
-quadrant: 1
+quadrant: 3
 ring : 0
 moved: 1
 ---
 
 # PostgreSQL
```
Comment (abestel) [inline docs/data/azure-api-management/azure-api-management.md:8]: The title should be changed

File: docs/data/express.js/express.js.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-label: Express.JS
+label: Express.js
 quadrant: 2
-ring : 1
-moved: 0
+ring : 0
+moved: 1
 ---
 
 # Express.JS
```
Comment (abestel) [inline docs/data/express.js/express.js.md:8]:
```suggestion
# Express.js
```

File: docs/data/azure-api-management/azure-api-management.md (hunk 1 above)
Comment (abestel) [inline docs/data/azure-api-management/azure-api-management.md:8]:
```suggestion
# Azure API Management
```

File: docs/data/kubernetes/kubernetes.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-label: Kubernetes
-quadrant: 1
+label: AKS
+quadrant: 3
 ring : 0
 moved: 0
 ---
 
-# Kubernetes
+# Azure Kubernetes Service (AKS)
```
Comment (abestel) [inline docs/data/kubernetes/kubernetes.md:8]: I'd leave it with "Kubernetes", AKS is mentioned in the text.

File: docs/data/next.js/next.js.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 ---
 label: Next.JS
 quadrant: 2
-ring : 2
-moved: 0
+ring : 0
+moved: 1
 ---
 
 # Next.JS
```
Comment (abestel) [inline docs/data/next.js/next.js.md:8]:
```suggestion
# Next.js
```

File: docs/data/express.js/express.js.md (hunk 2 above)
Comment (afreire-laptop) [reply docs/data/express.js/express.js.md]: done!

File: docs/data/azure-api-management/azure-api-management.md (hunk 1 above)
Comment (afreire-laptop) [reply docs/data/azure-api-management/azure-api-management.md]: done!

File: docs/data/next.js/next.js.md (hunk 4 above)
Comment (afreire-laptop) [reply docs/data/next.js/next.js.md]: Done!

**Review by abestel — APPROVED** [review]
Go for it

**Pull request merged**